modules/core/src/smithy4s/schema/Schema.scala

```diff
@@ -86,6 +89,31 @@ sealed trait Schema[A]{
     case document => Document.Decoder.fromSchema(this).decode(document).toOption
   }
 
+  /**
+    * When applied on a structure schema, creates a schema that, when compiled into
+    * a codec, will only encode/decode a subset of the data, based on the hints
+    * of each field.
+    *
+    * This can be used to only encode some fields of the data into the http body
+    *
+    * Returns a PartialSchema that indicates whether :
```

Comment:
**issue:** old name

Reply:
Addressed in https://github.com/disneystreaming/smithy4s/pull/875/commits/9cb8c6bde034946d6d0332034e385716cf15b9ee